### PR TITLE
Initial support for linking to off-budget accounts

### PR DESCRIPTION
**Pull request by psybers**
This adds a new option 'Create new account (off-budget)' when linking a bank account.

TODO:
- [x] have the list of unlinked accounts also list the off-budget accounts

### CODE REVIEW

**Comment by psybers** [comment]
image

**Review by youngcw — DISMISSED** [review]

File: packages/loot-core/src/server/main.ts [hunk 1]
```diff
@@ -620,6 +621,7 @@ handlers['gocardless-accounts-link'] = async function ({
       name: account.name,
       official_name: account.official_name,
       bank: bank.id,
+      offbudget: offBudget ? 1 : 0,
```
[the remark this answers is not in the document]
Comment (psybers) [reply packages/loot-core/src/server/main.ts:624]: I don't know. I have no GoCardless account to test things.

**Comment by youngcw** [comment]
@psybers Could you fix the conflicts?

@MatissJanis Can you confirm that GoCardless still works with this?  I imagine it does, but I don't want to merge in a possible bug just before the release.

**Comment by psybers** [comment]
@youngcw Conflicts resolved.

**Comment by MatissJanis** [comment]
Can confirm: GoCardless works as expected 👍 
Screenshot 2024-05-30 at 20 03 22

**Review by youngcw — APPROVED** [review]

**Comment by DenisBY** [comment]
Looks like it should resolve my feature request https://github.com/actualbudget/actual/issues/2444

**Pull request merged**